Formula/neovim-qt.rb

```diff
@@ -0,0 +1,79 @@
+class NeovimQt < Formula
+  desc "Neovim GUI, in Qt5"
+  homepage "https://github.com/equalsraf/neovim-qt"
+  url "https://github.com/equalsraf/neovim-qt/archive/v0.2.16.1.tar.gz"
+  sha256 "971d4597b40df2756b313afe1996f07915643e8bf10efe416b64cc337e4faf2a"
+  license "ISC"
+  head "https://github.com/equalsraf/neovim-qt.git", branch: "master"
+
+  depends_on "cmake" => :build
+  depends_on "neovim-remote" => :test
+  depends_on "neovim"
+  depends_on "qt@5"
+
+  on_linux do
+    depends_on "gcc"
+  end
+
+  fails_with gcc: "5" # Qt needs GCC
+
+  def install
+    system "cmake", "-S", ".", "-B", "build", *std_cmake_args, "-DUSE_SYSTEM_MSGPACK=ON"
+    system "cmake", "--build", "build"
+
+    if OS.mac?
+      prefix.install "build/bin/nvim-qt.app"
+      bin.install_symlink prefix/"nvim-qt.app/Contents/MacOS/nvim-qt"
+    else
+      bin.install "build/bin/nvim-qt"
+    end
```

Comment:
We should check if there's actually an `install` target we can use, rather than just manually installing things ourselves.

Reply:
Yes, there is an `install` target. It is part of the Makefile that's generated by Cmake, and it uses `"-DCMAKE_INSTALL_PREFIX=#{prefix)"`, which appears to be [one of the `std_cmake_args`](https://rubydoc.brew.sh/Formula.html#std_cmake_args-instance_method).

Out of curiosity, why did you switch from `make` to `cmake --build build`? Is it OK if I reinstate the `mkdir "build" do` block so we can use the `make` and `make install`?

Reply:
You can do the equivalent of `make install` with `cmake --install build`.

Reply:
@jgehrig sorry for my lack of CMake knowledge. What would the right `cmake --install` invocation be in this case? `cmake -S . --install`?

Reply:
```
cmake --install build
```

Reply:
Thanks @carlocab . `cmake --install build` causes the Mac `.app` bundle to be installed to `bin/`, which causes an warning in `brew audit`. Is that OK?

It would be nice if we could install the bundle to `/Applications`. Maybe then we need a Cask instead of a Formula?

Reply:
I see; we can just move things around after the fact. The same thing would be needed with `make install` anyway. Formulae cannot install outside of `HOMEBREW_PREFIX`, so we can't install anything into `/Applications`. We need a cask to do something like that.

Reply:
Noted! Maybe I can help the Nvim-Qt project set up a pipeline to automatically build and upload the app bundle.

Reply:
Yes, that would be nice, and probably a much better experience for macOS users.